source/features/selection-in-new-tab.tsx

```diff
@@ -2,22 +2,24 @@ import select from 'select-dom';
 import onetime from 'onetime';
 
 import features from '.';
-import {isEditable} from '../helpers/dom-utils';
+import registerHotkey from '../github-helpers/register-hotkey';
 
-function openInNewTab({key, target}: KeyboardEvent): void {
+function openInNewTab(): void {
 	const selected = select('.navigation-focus a.js-navigation-open[href]');
-	if (selected && key === 'O' && !isEditable(target)) {
-		void browser.runtime.sendMessage({
-			openUrls: [selected.href],
-		});
-
-		// Get the list element that contains the unread class and mark it as read.
-		selected.closest('.unread')?.classList.replace('unread', 'read');
+	if (!selected) {
+		return;
 	}
+
+	void browser.runtime.sendMessage({
+		openUrls: [selected.href],
+	});
+
+	// Get the list element that contains the unread class and mark it as read.
+	selected.closest('.unread')?.classList.replace('unread', 'read');
 }
 
 function init(): void {
-	document.addEventListener('keypress', openInNewTab);
+	registerHotkey('O', openInNewTab);
```

Comment:
https://user-images.githubusercontent.com/46634000/163721843-122ed089-e69e-4105-87a0-4473b65c0ae8.mp4

Reply:
I think it would be shift-o

Reply:
> I think it would be shift-o

No, GitHub understand `O` just fine. We already have `S` for the "Select notification" dropdown, for example.